app/controllers/activities_controller.rb

```diff
@@ -0,0 +1,36 @@
+class ActivitiesController < ApplicationController
+	include ActivitiesHelper
+	before_filter :redirect_to_root_if_not_logged_in
+	before_filter :redirect_to_root_if_not_admin, only: :dashboard
+	before_action :set_todays_activity, except: :dashboard
+
+	def index
+		@activities = current_user.get_authorized_activities(user_id: params['user'])
```

Comment:
Something is wrong. You are calling on a user object and also passing a user.
Only one should happen.